doc/source/examples/03_modeling/export_design.mystnb

```diff
@@ -0,0 +1,164 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Exporting designs
+
+After a design has been created, users are typically interested in using bringing
+the design into a CAD tool for further development. This notebook demonstrates
+how to export a design to the various supported CAD formats.
+
+## Creating a design
+
+A simple design is created for demonstration purposes. This design consists of
+a set of rectangular pads with a circular hole in the center.
+
+```{code-cell} ipython3
+from ansys.geometry.core import Modeler
+from ansys.geometry.core.math import UNITVECTOR3D_X, UNITVECTOR3D_Y, Point2D
+from ansys.geometry.core.sketch import Sketch
+
+# Instantiate the modeler - this will connect to the service
+modeler = Modeler()
+
+# Create a new design
+design = modeler.create_design("ExportDesignExample")
+
+# Create a new sketch
+sketch = Sketch()
+
+# A simple rectangle will be created
+sketch.box(Point2D([0, 0]), 10, 5)
+
+# The sketch will be extruded and the resulting body will be displaced
+# in order to make a plane of rectangles
+for x_step in [-60, -45, -30, -15, 0, 15, 30, 45, 60]:
+    for y_step in [-40, -30, -20, -10, 0, 10, 20, 30, 40]:
+        # Extrude the sketch
+        body = design.extrude_sketch(f"Body_X_{x_step}_Y_{y_step}", sketch, 5)
+
+        # Displace the body in the x and y directions
+        body.translate(UNITVECTOR3D_X, x_step)
+        body.translate(UNITVECTOR3D_Y, y_step)
+
+# Plot the design
+design.plot()
+```
+
+## Exporting the design
+
+The design can be exported to various CAD formats. The supported formats are
```

Comment:
This section is all in passive voice. If you write with "you" it will much cleared. "You can export the design to various.... For the formats supported, see [DesignFileFormat]...."

Reply:
Yeah - common error on my side. I tend to use the passive voice to avoid the Vale warnings regarding the usage of "you" and "we". Let me rewrite it somehow.